### PR TITLE
Create Oof Reply

### PR DESCRIPTION
**Pull request by RDIL**
all oofs are met with another oof
lol imagine like 50 hyperium users all in 1 lobby together and one says oof xD

### CODE REVIEW

File: src/main/java/cc/hyperium/integrations/oofreply.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package cc.hyperium.integrations.oofreply;
```
Comment (asbyth) [inline src/main/java/cc/hyperium/integrations/oofreply.java]: java uses capitalized class-names btw

Comment (RDIL) [reply src/main/java/cc/hyperium/integrations/oofreply.java]: is it just a linting thing or required?

Comment (RDIL) [reply src/main/java/cc/hyperium/integrations/oofreply.java]: I think its fixed now.

Comment (RDIL) [reply src/main/java/cc/hyperium/integrations/oofreply.java]: python doesnt :p

**Comment by RDIL** [comment]
Maybe I'll make this into an addon sometime in the future.  That would be fun.

**Pull request closed without merging**